include/http.h

```diff
@@ -73,6 +73,11 @@ typedef struct conn_t
   uv_stream_t *handle;
   boolean ws_handshake_sent;
   uv_stream_t *proxy_handle;
+  enum connection_type_t
```

Comment:
why do you have here `enum connection_type_t { ... } type;` instead of just
```c
enum
{
  TYPE_REQUEST,
  TYPE_WEBSOCKET
} type;
```

I don't understand the meaning of `connection_type_t` here.

Reply:
You are right, I'll fix it.